config/rbac/role.yaml

```diff
@@ -128,6 +128,16 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
```

Comment:
This can be removed as the changes are contained in PR #205 so we can look into merging that in first

Reply:
The code is autogenerated, so lets hang onto it for now. 

If #205 gets merged first, a simple rebase should clear up this diff. If this gets merged first, the logic here won't represent a "partial" solution and instead contain everything needed to get clusterurlmonitors working for hypershift :slightly_smiling_face: